src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx

```diff
@@ -3,7 +3,7 @@ export const description = "This guide describes how to query the MM2 SQLite dat
 
 # How to Query the MM2 SQLite Database
 
-The Komodo DeFi Framework API stores historical information such as swaps and orders within an SQLite database, located under the DB user data folder with each wallet having its own subfolder represented by a hexideciaml string.
+The Komodo DeFi Framework API stores historical information such as swaps and orders within an SQLite database, located under the DB user data folder with each wallet having its own subfolder represented by a hexidecimal string.
```

Comment:
hexadecimal

Reply:
fixed in https://github.com/KomodoPlatform/komodo-docs-mdx/pull/364/commits/d19dff74af377891bce520597742c7dbadd22658